scripts/generate_ignition_import_config.py

```diff
@@ -0,0 +1,167 @@
+        PackageGroup('ignition-math6'),
+        PackageGroup('ignition-msgs5'),
+        PackageGroup('ignition-physics2'),
+        PackageGroup('ignition-plugin'),
+        PackageGroup('ignition-rendering3'),
+        PackageGroup('ignition-sensors3'),
+        PackageGroup('ignition-tools'),
+        PackageGroup('ignition-transport8'),
+        PackageGroup('ogre-2.1'),
+        PackageGroup('sdformat9'),
+    ],
+    'edifice': [
+        PackageGroup(f'ignition-edifice'),
+        PackageGroup('ignition-cmake2'),
+        PackageGroup('ignition-common4'),
+        PackageGroup('ignition-fuel-tools6'),
+        PackageGroup('ignition-gazebo5'),
+        PackageGroup('ignition-gui5'),
+        PackageGroup('ignition-launch4'),
+        PackageGroup('ignition-math6'),
+        PackageGroup('ignition-msgs7'),
+        PackageGroup('ignition-physics4'),
+        PackageGroup('ignition-rendering5'),
+        PackageGroup('ignition-sensors5'),
+        PackageGroup('ignition-tools'),
+        PackageGroup('ignition-transport10'),
+        PackageGroup('ignition-utils1', skip_packages=['libignition-utils-dev']),
+        PackageGroup('ogre-2.2'),
+        PackageGroup('sdformat11'),
+    ],
+    'fortress': [
+        PackageGroup('ignition-fortress'),
+        PackageGroup('ignition-cmake2'),
+        PackageGroup('ignition-common4'),
+        PackageGroup('ignition-fuel-tools7'),
+        PackageGroup('ignition-gazebo6'),
+        PackageGroup('ignition-gui6'),
+        PackageGroup('ignition-launch5'),
+        PackageGroup('ignition-math6'),
+        PackageGroup('ignition-msgs8'),
+        PackageGroup('ignition-physics5'),
+        PackageGroup('ignition-plugin'),
+        PackageGroup('ignition-rendering6'),
+        PackageGroup('ignition-sensors6'),
+        PackageGroup('ignition-tools'),
+        PackageGroup('ignition-transport11'),
+        PackageGroup('ignition-utils1', skip_packages=['libignition-utils-dev']),
+        PackageGroup('ogre-2.2'),
+        PackageGroup('sdformat12'),
+    ]
+}
+
+package_groups = PACKAGE_GROUPS[args.ignition_suite]
+
+target_repo = f'http://packages.osrfoundation.org/gazebo/{args.os}-stable'
+
+resp = urllib.request.urlopen(f'{target_repo}/dists/{args.suite}/main/binary-amd64/Packages')
+packages_file = PackagesFile(resp.read().decode())
+
+version_spec_re = re.compile('(?P<version>[^-]+)-(?P<inc>\d+)(?P<rest>.*)')
+for group in package_groups:
+    expected_group_version = None
+    group.packages = [pkg.name for pkg in packages_file.packages.values() if pkg.source_package == group.source_package and pkg.name not in group.skip_packages]
+    for p in group.packages:
+        if not expected_group_version:
+            expected_group_version = packages_file.packages[p].version
+        elif packages_file.packages[p].version != expected_group_version:
+            print(f'{p} in {group.packages[0]} does not match expected version {expected_group_version}', file=sys.stderr)
```

Comment:
The check is nice to have. However It can detect problems with versions for orphaned packages, like the bad named `libgz-math6-eigen-dev` now renamed to `libgz-math6-eigen3-dev`. If we keep the exit after the check, the script can not continue.

My proposal is to remove the exit and leave the warning goes into stderr which does not affect standard redirections and diffs.